cavalry/db_django2.py

```diff
@@ -37,36 +38,17 @@ def executemany(self, sql, param_list):
             self._record(sql, param_list, duration, times)
 
     def _record(self, sql: str, params, duration: float, times=None):
```

Comment:
Why not use `record` directly on line 38?

Reply:
I wanted to keep the diff minimal; `self._record()` is used in two places (`execute` and `executemany`).

Reply:
Fair enough!